examples/vmm/vmm-dev.mk

```diff
@@ -124,3 +128,11 @@ clobber:: clean
 
 # How to build libvmm.a
 include ${LIBVMM}/vmm.mk
+
+# Add an extra dependency for libvmm.a
+# Note: this should be in ${LIBVMM}/vmm.mk
+libvmm/virtio/block.o: ${SDDF}/include
```

Comment:
I noticed when working on libvmm that updating the headers did not cause objects to be rebuilt, I think we should add this as well as `libvmm/include` to `libvmm/vmm.mk` in another PR.

Reply:
libvmm/include should not be necessary providing -MD is in CFLAGS.

Reply:
I've just looked at vmm.mk --- it has a different bug, stemming from when object files were put in a libvmm directory, not a src directory -- the result is the dependency files that track headers were never included.
PR for libvmm coming soon.

Reply:
Still seems weird that we have this line. Do we need it anymore after the fix to libvmm?